k8s/namespaces/probate/probate-cron-smee-and-ford-extract/prod-01.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: probate-cron-smee-and-ford-extract
+spec:
+  releaseName: probate-cron-smee-and-ford-extract
+  values:
+    job:
+      concurrencyPolicy: Forbid
```

Comment:
I would recommend having a `prod.yaml` file and set everything except cron expression in there to avoid duplication of all env vars. You need to patch it in  https://github.com/hmcts/cnp-flux-config/pull/17709/files#diff-bac4f65910225fbbee02d7e88934250e89c3a588df639bd4bfa51c4490f4d134R7 after adding it

Reply:
Thanks @adusumillipraveen, this job is already on prod, here we are adding for pod 01. Can we raise separate ticket to do this using common prod.yml please?

Reply:
the element of duplication comes only because you are adding prod-01. If not, there was no duplication. So, i would recommend adding prod.yml with this PR pls.

Reply:
Done the changes with help from @damongreen123. @adusumillipraveen can you please review it

Reply:
@adusumillipraveen can you please review it as we have to release it today on prod